lib/actions/FrontendAction.js

```diff
@@ -26,24 +26,30 @@ class FrontendAction {
 
   /**
    * Registers the frontend command.
-   * @param {Command} commander Instance of the commander module.
+   * @param {Command} caporal Instance of the commander module.
    */
-  register (commander) {
-    commander
-      .command('frontend <action>')
-      .description('The frontend action to run. Possible actions are \'start\' and \'setup\'.')
+  register (caporal) {
+    caporal
+      .command('frontend start')
+      .description('Start the webpack for the frontend development')
       .option('-t, --theme [value]', 'The name of the theme that you want to be compiled by webpack.')
       .option('-h, --host [value]', 'The URL or IP address of your local development environment.')
       .option('-p, --port [value]', 'The port to use for accessing the output via browser.')
-      .action(this.run.bind(this))
+      .action(this.run.bind(this, 'start'))
+
+    caporal
+      .command('frontend setup')
+      .description('Change your settings for the frontend development')
```

Comment:
Common style of these textes were what the command does, not what you con do with it. My suggestion would be 'Changes the settings for the frontend development server'